src/views/List.jsx

```diff
@@ -22,6 +26,36 @@ export function List({ data, listId }) {
 		);
 	};
 
+	const CopyToken = () => {
+		const [copied, setCopied] = useState(false);
+
+		const handleCopyToClipboard = () => {
+			try {
+				copy(listId);
+				setCopied(true);
+
+				setTimeout(() => {
+					setCopied(false);
+				}, 3000);
+			} catch (err) {
+				console.error('Copy failed:', err);
+			}
+		};
+
+		return (
+			<div>
+				<span>
+					Use the token <em>{listId}</em>
+					<button onClick={handleCopyToClipboard}>
```

Comment:
I wonder if we should just say: " List Toke: _novo soon snag_ [📋](https://emojis.wiki/clipboard/)"

Reply:
I'm not sure if we need to use try and catch here. Do you think about the browser that doesn't allow copy?

Reply:
I'll simplify the message - I wasn't sure what to write for the message 😅

Reply:
What do you guys think about copy-clipboard? It's supported by most modern web browsers (Chrome, Firefox, Edge, Safari, Brave).

Reply:
I just checked the library. It's a bit weird.

This is their example. 
```const copy = require('clipboard-copy')
 
button.addEventListener('click', function () {
  copy('This is some cool text')
})```

If the function returns a promise(it does) we need "try and error", also we need to add await for the promise to resolve or reject. 

I also found this built-in function. It achieves the same.
```const handleCopyToClipboard = () => {
			navigator.clipboard
				.writeText(listId)
				.then(() => {
					setCopied(true);
					setTimeout(() => {
						setCopied(false);
					}, 3000);
				})
				.catch((err) => {
					console.log(err.message);
				});
		};```
 https://stackoverflow.com/questions/74056561/copying-to-clipboard-react#:~:text=The%20reason%20why,elements%20manually%20instead

Anyway, you can go ahead as you want.

Reply:
I just added the async/await keywords to the function. I'm fine using the built-in function too. Thank you for finding that! How does everyone feel about using `copy` vs. the built-in?